measure_cocoapod_size.py

```diff
@@ -79,6 +81,12 @@ def InstallPods(cocoapods, target_dir, spec_repos, target_name, mode):
     for pod, version in cocoapods.items():
       if version:
         podfile.write(' pod \'{}\', \'{}\'\n'.format(pod, version))
+      elif pod_sources is not None and pod in pod_sources.keys():
+        # pod_sources[pod] should have pairs like:
+        # "git":"sdk/repo.git", "branch":"main" or
+        # "path":"~/Documents/SDKrepo"
+        pod_source_config =  ", ".join(map(lambda x: ":{} => \'{}\'".format(x[0], x[1]), pod_sources[pod].iteritems()))
```

Comment:
I believe `iteritems` doesn't work on dictionaries in python 3.x. Curious, how does this library choose which version of python to use and can enforce a specific version (or range of versions)? 
If it will be used on  python 3.x then I think we should use `items`.

Reply:
It depends on the following line.
https://github.com/google/cocoapods-size/blob/master/measure_cocoapod_size.py#L1

The tool here seems to be using pyhthon 2. Have updated it to `items`.